bifacial_radiance/main.py

```diff
@@ -3975,7 +3939,8 @@ def _set1axis(self, cumulativesky=True, axis_azimuth=180, limit_angle=45,
                                                axis_tilt = axis_tilt,
                                                backtrack = backtrack,
                                                gcr = gcr,
-                                               fixed_tilt_angle=fixed_tilt_angle)
+                                               fixed_tilt_angle=fixed_tilt_angle,
+                                               fixed_tilt_azimuth=fixed_tilt_azimuth)
```

Comment:
@shirubana  - we're already using the axis_azimuth to be the array azimuth orientation whether it's fixed tilt or tracked, why is there need for another variable?

Reply:
Well, when I tested it it was not working properly. Then I saw your documentation and what was supposed to happen but I had already changed it, sorry..  Something was just not working righ.

Have you seen my notes in the project? I did comment on this or maybe on the text and the below issue you brought up.

Reply:
I've put together pull request #330 that reverts this, if you want to review it.